src/components/Post/Post.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Post.css';
+import { User } from '../User/User';
+import { CommentList } from '../CommentList/CommentList';
+
+type PostProps = {
+  post: Post;
+};
+
+export const Post: React.FC<PostProps> = (props) => {
```

Comment:
```suggestion
interface Props = {
  post: Post;
};

export const Post: React.FC<Props> = (props) => {
```

исправь везде соответственно